hooks/advocate-conversion-utils.ts

```diff
@@ -28,7 +29,12 @@ async function fetchAdvocates (apiKey: string): Promise<Advocate[]> {
   }).eachPage((records, nextPage) => {
     for (const record of records) {
       const advocate = convertToAdvocate(record)
-      advocates.push(advocate)
+      // only commit to store, advocates
+      // who have filled out the approval form
+      // and provided their Slack ID
```

Comment:
As per spec, a multi-line comment should be written like this:

```suggestion
      /**
       * only commit to store, advocates who have filled out the approval form
       * and provided their Slack ID
       */
```

https://www.ecma-international.org/ecma-262/5.1/#sec-7.4